src/status_im/multiaccounts/login/core.cljs

```diff
@@ -106,6 +106,7 @@
    (wallet/initialize-tokens custom-tokens)
    (wallet/initialize-favourites favourites)
    (wallet/get-pending-transactions)
+   (wallet/get-opensea-collections)
```

Comment:
lets at least rename it in clojure, because here we don't care if its opensea or whatever else